apps/designer/app/canvas/features/wrapper-component/wrapper-component.tsx

```diff
@@ -60,6 +63,21 @@ export const WrapperComponentDev = ({
   const [selectedInstance] = useSelectedInstance();
   const [, setSelectedElement] = useSelectedElement();
 
+  const [allUserProps] = useAllUserProps();
+  const instanceProps = allUserProps[instance.id];
```

Comment:
what the reason of removing useUserProp?

Reply:
useUserProp gives us only allProps[instanceId].props. But we need to send to designer allProps[instanceId] with additional info